### ISSUE TITLE
Missing W2UI in http://reactive-table.meteor.com/

### ISSUE THREAD

**Issue by pdavidow** [aslagle/reactive-table#210]
http://w2ui.com/web/home

**Comment by aslagle** [comment]
Yes, this definitely needs an update. aldeed:tabular is also missing.

I'd appreciate a PR - the code is here: https://github.com/aslagle/reactive-table/blob/master/examples/table-features/table-features.js#L68

**Comment by pdavidow** [comment]
I am not comfortable doing the update/PR.


On Sun, May 10, 2015, at 03:26 PM, Amy Slagle wrote:
> Yes, this definitely needs an update. aldeed:tabular is also missing.


> I'd appreciate a PR - the code is here:
> https://github.com/aslagle/reactive-table/blob/master/examples/table-features/table-features.js#L68


> — Reply to this email directly or view it on GitHub[1].

>


Links:

  1. https://github.com/aslagle/reactive-table/issues/210#issuecomment-100685208
